examples/user_guide/Geometry_Data.ipynb

```diff
@@ -237,7 +241,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "poly.split(datatype='array')"
+    "poly.split(datatype='columns')"
```

Comment:
Does the markdown above now need to be updated? Currently it is:

> Using the ``datatype`` argument the data may instead be returned in the desired format, e.g. a list of arrays:

Reply:
Ah, that was on my todo list but seems I forgot.

Reply:
I generally wouldn't recommend using array here since you can't represent the additional holes  that way. Hence why I switched.

Reply:
I'm fine with the switch as long as the explanation matches.